app/components/events/card_component.html.erb

```diff
@@ -1,4 +1,5 @@
-<li class="category__nav-card">
+<!-- This <li> element is dynamically generated within a parent template that includes the surrounding <ul>. -->
+<!-- //NOSONAR --><li class="category__nav-card">
```

Comment:
```suggestion
<li class="category__nav-card">
```
I'd imagine with the new comment that this trigger could go off naturally. I'm not sure though